learning_observer/learning_observer/auth/social_sso.py

```diff
@@ -143,6 +159,28 @@ async def social_handler(request):
     return aiohttp.web.HTTPFound(url)
 
 
+async def determine_roster_source(request):
```

Comment:
Remove this function.

Reply:
I had suggested we include this function. In the future, we will likely need a few more lines of code to dig into the request for additional information to pass into pmss, such as what school the user is from.

Reply:
I agree that in the future, we will likely need this function. However, as a matter of process, I think it makes sense to break out that function at the time we need it (e.g. in the future). I tend to leave something like this is a comment: `# Break out into a function when we need X Y and Z`

@JohnDamilola You have a difference of opinion between reviewers. Your call.

Reply:
Removed the function for now until there is more functionality for the function